### PR TITLE
Net Beans Cheat sheets

### PR DESCRIPTION
**Pull request by mahavirrathod**
What does your Instant Answer do?

Provides all the shortcuts of netbeans IDE

What is the data source for your Instant Answer? (Provide a link if possible)

https://netbeans.org/features/index.html

Why did you choose this data source?

It provides best explaination about netbeans

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)

Programmer, developer, coders

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)
netbeans

### CODE REVIEW

**Comment by daxtheduck** [comment]
@mahavirrathod Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by mintsoft** [comment]
@mahavirrathod Can you add the link to the IA page into your PR?

**Comment by MariagraziaAlastra** [comment]
Hey @mahavirrathod, thanks for this!
As @mintsoft said, could you create an IA Page [here](https://duck.co/ia/new_ia) and then paste the link to it in your first comment? 
Thanks!

File: share/goodie/cheat_sheets/json/netbeans.json [hunk 1]
```diff
@@ -0,0 +1,364 @@
+{  
+   "id":"netbeans_cheat_sheet",
+   "name":"Netbeans IDE",
+   "description":"NetBeans is an open-source integrated development environment (IDE) for developing with Java, PHP, C++, and other programming languages. ",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/netbeans.json]: @mahavirrathod Lets have a shorter description preferably about the contnets of this cheat sheet

File: share/goodie/cheat_sheets/json/netbeans.json [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+   "metadata":{  
+      "sourceName":"Shortcut World",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/netbeans.json]: @mahavirrathod Lets chaneg this to "NetBeans"

**Comment by jdorweiler** [comment]
This pull request is being closed due to inactivity but, if anyone would like to continue it, please @ mention 'duckduckhack-contributors' for help and http://docs.duckduckhack.com/submitting/submitting-overview.html for reference.

**Pull request closed without merging**